src/Polygon2d.elm

```diff
@@ -261,6 +261,28 @@ convexHull points =
     singleLoop (lower ++ upper)
 
 
+{-| Constructs a regular convex polygon with the specified center point, circumradius and number of sides.
+-}
+regular : Point2d units coordinates -> Quantity Float units -> Int -> Polygon2d units coordinates
+regular center radius sides =
```

Comment:
Probably bikeshedding but I wonder about putting `sides` first...kinda nicely readable as (e.g.) "a regular 12-sided polygon" and I think it's likely to be the simplest argument (often probably a hardcoded constant) so it's kind of nice to read that first before you dive into deciphering any logic about how center point or radius have been computed.

The only irritating thing is that I'm not sure I'd want the number of sides as the first argument if we also have `fromCircumcircle` and `fromIncircle`, since those seem like they should probably have the circle argument first...but maybe that slight inconsistency is OK.

Reply:
Hah I put it last since in my use case the radius and center point were constant, but the number of sides was dynamic 😆 

So you want `sides center radius`?

Reply:
That's what I was thinking, yeah...I guess I was picturing the common use case being "I want to draw a pentagon" or similar. But if dynamic number of sides is a common use case (which also does seem plausible, for use cases like yours and others) then maybe it's best leaving `regular` as is and then also have `sides` as the last argument to `fromCircumcircle` and `fromIncircle` functions.

How about let's leave it as is for now, and then I might mull this over a bit more before I push out a release 🙂